tests/unit/storage/data/external/ExternalClients.js

```diff
@@ -154,6 +164,45 @@ describe('external backend clients', () => {
                     });
             });
         });
+
+        if (backend.config.type !== 'azure') {
+            it(`${backend.name} objectPutTagging() then objectDeleteTagging() should set tags then delete it`, done => {
+                const key = {
+                    key: 'externalBackendTestBucket/externalBackendTestKey',
+                    dataStoreName: backend.config.dataStoreName,
+                    response: new stream.PassThrough(),
+                };
+                const bucket = new DummyBucket(backend.config.bucketName);
+                const objectMd = {
+                    tags: {
+                        Key1: 'value_1',
+                        Key2: 'value_2',
+                    },
+                    location: [
+                        {
+                            dataStoreVersionId: 'latestversion',
+                        },
+                    ],
+                };
+                testClient.objectPutTagging(
+                    key,
+                    bucket,
+                    objectMd,
+                    log,
+                    assert.ifError,
+                );
+                testClient.objectDeleteTagging(
+                    key,
+                    bucket,
+                    objectMd,
+                    log,
+                    (err) => {
+                        assert.ifError(err);
+                        done();
+                    },
```

Comment:
You can pass in done as the callback and it will do the same thing, https://mochajs.org/#asynchronous-code